### PR TITLE
Pre-release merge for arXiv base v0.17.1

### PR DESCRIPTION
**Pull request by mhl10**
* adds physics category descriptions (ARXIVOPS-1433)

### CODE REVIEW

File: arxiv/taxonomy/definitions.py [hunk 1]
```diff
@@ -1528,72 +1578,142 @@
...
         'is_general': False
     },
     'physics.bio-ph': {
         'name': 'Biological Physics',
         'in_archive': 'physics',
+        'description': 'Molecular biophysics, cellular biophysics, neurological '
+                       'biophysics, membrane biophysics, single-molecule '
+                       'biophysics, ecological biophysics, quantum phenomena in '
+                       'biological systems (quantum biophysics), theoretical '
+                       'biophysics, molecular dynamics / modelling and '
```
Comment (mhl10) [inline arxiv/taxonomy/definitions.py:1641]:
```suggestion
                       'biophysics, molecular dynamics/modeling and '
```

File: arxiv/taxonomy/definitions.py [hunk 2]
```diff
@@ -1528,72 +1578,142 @@
...
         'is_general': False
     },
     'physics.atm-clus': {
         'name': 'Atomic and Molecular Clusters',
         'in_archive': 'physics',
+        'description': 'Atomic and molecular clusters, nanoparticles: geometric, '
+                       'electronic, optical, chemical, magnetic properties, '
+                       'shell structure, phase transitions, optical spectroscopy, '
+                       'mass spectrometry, photoelectron spectroscopy, '
+                       'ionization potential,electron affinity, interaction '
```
Comment (mhl10) [inline arxiv/taxonomy/definitions.py:1615]:
```suggestion
                       'ionization potential, electron affinity, interaction '
```

**Review by jimentwood — APPROVED** [review]
looks good

**Review by SBBCornell — APPROVED** [review]
woohoo! Its great to have these category descriptions added, and they look good.

**Pull request merged**